kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -35,6 +35,7 @@ In the comments next to each cell, we've marked which component of the YellowPap
         <exit-code exit=""> 1 </exit-code>
         <mode> $MODE:Mode </mode>
         <schedule> $SCHEDULE:Schedule </schedule>
+        <scheduleTuple> getSchedule($SCHEDULE:Schedule) </scheduleTuple>
```

Comment:
Can we drop the `<schedule>` cell and have only the `<scheduleTuple>`?

Reply:
No. Schedule is still used in some places, for example when calling precompiled contracts.